install/terraform/open-match-build/README.md

```diff
@@ -11,6 +11,17 @@ a migration or emergency.
 If you're making changes to these files you must check in the .tfstate file as
 well as comment the reason why you're enabling a feature or making a change.
 
+## GCP Service Account Setup
+To use the terraform templates when developing Open Match, you need to have a [GCP service account](https://cloud.google.com/docs/authentication/getting-started) associated with your Open Match project and generate a key file. The terraform templates would read the credentials from `./cred.json` file to authenticate with GCP.
```

Comment:
I think pointing to this documentation is better:
https://www.terraform.io/docs/providers/google/provider_reference.html#credentials-1

Also we should remove creds.json from the file. It makes it harder to do automation.